backend/config/config.go

```diff
@@ -94,6 +97,7 @@ type Config struct {
 	NomadSkipVerify  bool
 	NomadHideEnvData bool
 	NomadAllowStale  bool
+	NomadDisableFS   bool
```

Comment:
just noticed we currently have `NomadAllowStale` so this should probably be `NomadAllowFS` to keep the same always positive/always negative wording flow